src/main/java/uk/gov/hmcts/cmc/claimstore/healthcheck/LaunchDarklyHealthIndicator.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.claimstore.healthcheck;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.launchdarkly.LaunchDarklyClient;
+
+@Component
+public class LaunchDarklyHealthIndicator implements HealthIndicator {
+    private static final String TOGGLE = "launch-darkly-health-roc-8014";
+    private final LaunchDarklyClient launchDarklyClient;
```

Comment:
do we need `roc-8014` in feature toggle name here?

Reply:
So this was part of the discussion I had with Chris and Pragnesh. We felt including the story name emphasises that it should be temporary, both in LD administration and in seeing it in the code. Feedback is welcome on it!

Reply:
trace-ability of ticket ->[ PR ]-> toggle ?